src/class/dfu/dfu_device.c

```diff
@@ -276,13 +277,15 @@ static void dfu_req_dnload_setup(uint8_t rhport, tusb_control_request_t const *
   // if they wish, there still will be the internal control buffer copy to this buffer
   // but this mode would provide zero copy from the class driver to the application
 
+  TU_VERIFY( request->wLength <= CFG_TUD_DFU_TRANSFER_BUFFER_SIZE);
```

Comment:
since this function return void, you need to add extra `,` to return void, by default TU_VERIFY return bool value. Please change it to

```
TU_VERIFY( request->wLength <= CFG_TUD_DFU_TRANSFER_BUFFER_SIZE, );
```